hvac/v1/__init__.py

```diff
@@ -691,6 +691,18 @@ def auth_ec2(self, pkcs7, nonce=None, role=None, use_token=True, mount_point='aw
 
         return self.auth('/v1/auth/{0}/login'.format(mount_point), json=params, use_token=use_token)
 
+    def auth_gcp(self, role, jwt, use_token=True):
+        """
+        POST /auth/gcp/login
```

Comment:
Would be nice to give this the full docstring treatment. Historically what you have here has been the model, but I've been trying to provide more information on newly added methods. E.g.: https://github.com/ianunruh/hvac/blob/master/hvac/v1/__init__.py#L674-L685

Reply:
Fixed. Let me if the doc string needs further changes.